### ISSUE TITLE
Unable to use arrow shortcuts in the popup time selector component using NVDA in Firefox or Chrome

### ISSUE THREAD

**Issue by yanmariomenev** [telerik/kendo-angular#4874]
### Description

It seems that the arrow key shortcuts and tabbing are not working with NVDA in the TimeSelector component. Tested on Windows with Firefox, Chrome, and the latest version of NVDA at the time (2025.3.3).  Tabbing seems not to work well, but it still focuses some elements in the popup.

When fixing the arrow key navigation, the screen reader should also provide useful information when switching between hours, minutes, and am/pm. 

NVDA Focus mode was also tested, and in that case, nothing works, not even the input date changing.

### Steps To Reproduce

Use the [up and down](https://www.telerik.com/kendo-angular-ui-develop/components/dateinputs/timepicker/keyboard-navigation#navigation-scenarios) arrow navigation when the popup is opened and the TimeSelector is focused in the following [demo](https://www.telerik.com/kendo-angular-ui-develop/components/dateinputs/timepicker/keyboard-navigation#keyboard-navigation).

### Screenshots or video

https://github.com/user-attachments/assets/d1958faf-8932-424a-a972-ac43d157ed4b

https://github.com/user-attachments/assets/d704c5f3-8565-47f4-936f-bed606e69e25

## Workaround

https://stackblitz.com/edit/angular-qtmq5udi-n87zpkb6?file=src%2Fapp%2Fapp.component.ts
